css/styles.css

```diff
@@ -0,0 +1,811 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  border: 0 none;
+  list-style: none outside none;
+  outline: 0 none;
+}
+
+body {
+  overflow-x: hidden;
+  font-size: 15px;
+  font-family: "Roboto Condensed", sans-serif;
+  line-height: 18px;
+  font-weight: bold;
+  background-color: #ddd;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: black;
+  font-size: 20px;
+  line-height: 20px;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: bold;
+  margin-bottom: 10px;
+  overflow: hidden;
+}
+
+ul {
+  padding-top: 10px;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+  overflow: hidden;
+}
+
+img {
+  width: 100%;
+}
+
+.item-3,
+.item-2,
+.item-1 {
+  position: relative;
+  font: bold 14px/14px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+  display: inline-block;
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class name can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush: